packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -472,7 +472,7 @@ def vip_workload_test(dcos_api_session, container, vip_net, proxy_net, ipv6,
     # the network mode.
     # it is used for user network mode only
     if ipv6:
-        network_name = 'dcos6'
+        network_name = '{}6'.format(network_name)
```

Comment:
We may make a note here that calico ipv6 is not supported yet.

Reply:
Will add, however I have a feeling we might forget that comment after we add IPV6 support 😄